### DISCUSSION TITLE
Do we have Receiver in Windows developed in DotNet?

### DISCUSSION THREAD

**Discussion by panthamm** [open-telemetry/opentelemetry-collector#3087] [Q&A] [answered]
Do we have any receivers in windows developed in DotNet?

**Comment by bogdandrutu** [comment, marked as answer]
For dotnet or written in dotnet? Collector and all components are written in go. For dotnet I think there is something in contrib repo

**Reply by panthamm** [reply]
I am specifically looking for collectors written in .net. I don't see collectors written .net in contrib repo.

**Reply by bogdandrutu** [reply]
Why would you need to be written in ".net"? The whole collector is golang, so we probably not going to write .net components

**Reply by bogdandrutu** [reply]
You can run golang apps on windows.
